src/main/scala/com/twitter/diffy/lifter/JsonLifter.scala

```diff
@@ -25,7 +25,7 @@ object JsonLifter {
     case JsonToken.START_ARRAY        =>
       node.elements.toSeq.map {
         element => lift(element)
-      }  sortBy { _.toString }
```

Comment:
That's weird. Can you share a bit more about what the differences look like. By definition, if all you have are ordering differences then there should be no differences left after sorting. This might be a bug that requires a different fix.